files/root/menu

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
```

Comment:
Please move the script to a non-data directory.
Since it's a root-only command, `/usr/local/sbin` should be good or `/usr/libexec` if the script shouldn't be called manually.